lib/carto/user_creator.rb

```
@@ -2,7 +2,7 @@ module Carto
   class UserCreator
 
     # NOTE: copied from the superadmin users_controller.rb
-    def create(params)
+    def create!(params)
```

I've changed some of these calls so failures don't get unnoticed. The commands will now raise an error instead of failing silently, and the messages will be *unacked* instead of *acked*

---

Yeah, the `user.save` vs `user.save!` bothered me at the time.

Let's be prepared for those failures at deployment time :)

I'd like to clean up rollbar and the dead letter topic to hone our ability to detect real failures vs "not really a problem"